doc/developers/contributing.rst

```diff
@@ -743,13 +743,10 @@ and the old behavior will be removed in version 0.20.
 
 .. currentmodule:: sklearn
 
-Python 3.x support
-------------------
+Python versions supported
+-------------------------
 
-All scikit-learn code should work unchanged in both Python 2.[67]
-and 3.2 or newer. Since Python 3.x is not backwards compatible,
-that may require changes to code and it certainly requires testing
-on both 2.7 and 3.2 or newer.
+All scikit-learn code should work unchanged in both Python 2.7 and 3.4 or newer. Since Python 3.x is not backwards compatible, that may require changes to code and it certainly requires testing on both 2.7 and 3.4 or newer.
```

Comment:
Haha did you mean "I know that this is not required"?
Regardless, thanks for mentioning this! Noted and changed.